src/scripts/main.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// write code here
+const list = document.querySelectorAll('li');
```

Comment:
Consider giving a more descriptive name to the variable 'list', such as 'treeItems' or 'treeNodes' to indicate that it refers to the expandable tree elements. 🌳